setup.cfg

```diff
@@ -18,14 +18,29 @@ version = 0.6
 [options]
 python_requires = >=3.5
 install_requires =
-    numpy==1.13.3; python_version=='3.5' and platform_system!='AIX'
-    numpy==1.13.3; python_version=='3.6' and platform_system!='AIX'
-    numpy==1.14.5; python_version=='3.7' and platform_system!='AIX'
+    # NOTE: platform_system 'AIX' and platform_machine 'aarch64' are mutually
+    #       exclusive so they are specified by themself when setting their
+    #       numpy version.
+
+    # platform_system=='AIX' requires numpy 1.16.0 (needed for Python < 3.8)
     numpy==1.16.0; python_version=='3.5' and platform_system=='AIX'
     numpy==1.16.0; python_version=='3.6' and platform_system=='AIX'
     numpy==1.16.0; python_version=='3.7' and platform_system=='AIX'
-    numpy==1.17.3; python_version=='3.8'
+
+    # platform_machine=='aarch64' requires numpy 1.19.0 (needed for Python < 3.9)
+    numpy==1.19.0; python_version=='3.5' and platform_machine=='aarch64'
+    numpy==1.19.0; python_version=='3.6' and platform_machine=='aarch64'
+    numpy==1.19.0; python_version=='3.7' and platform_machine=='aarch64'
+    numpy==1.19.0; python_version=='3.8' and platform_machine=='aarch64'
```

Comment:
For python 3.6-3.8 we want `1.19.2` as the minimum I think, that fixed quite a nasty bug (see https://github.com/numpy/numpy/releases/tag/v1.19.2).

Reply:
I altered your suggested comment to include this